cmd/otelcol/Dockerfile

```diff
@@ -53,6 +60,10 @@ COPY --from=otelcol --chown=999 /migratecheckpoint /
 COPY --from=otelcol --chown=999 /otel /etc/otel
 COPY --from=otelcol --chown=999 /otel/collector /etc/otel/collector
 COPY --from=agent-bundle --chown=999 /usr/lib/splunk-otel-collector /usr/lib/splunk-otel-collector
+COPY --from=jmx --chown=999 /tmp/opentelemetry-jmx-metrics.jar /opt/opentelemetry-java-contrib-jmx-metrics.jar
+
+#COPY --from=otelcol --chown=999 /tmp /tmp
+#ENV TMPDIR=/tmp
```

Comment:
Let's remove this if it is no longer needed.
```suggestion
```

Reply:
Good call, forgot it there!